src/mprismetadata.cpp

```diff
@@ -743,21 +743,35 @@ QVariantMap MprisMetaData::extraFields() const
     for (auto c = priv->m_metaData.cbegin();
          c != priv->m_metaData.cend();
          ++c) {
-        if (c.key().count(':') == 1 && !c.key().startsWith(QLatin1String("mpris:")) && !c.key().startsWith(QLatin1String("xesam:"))) {
+        if (!converters.contains(c.key()) && c.key() != MetaFieldInternalYear && c.key().count(':') == 1) {
```

Comment:
`MetaFieldInternalYear` intentionally doesn't contain `:`. so the `count(':') == 1` implies `!= MetaFieldInternalYear`.

Reply:
Good idea, though I left a comment to explain why